### PR TITLE
man: fixes some man pages

### PR DESCRIPTION
**Pull request by jeromemarchand**
In the man pages of tcplife and tcpstates, the name of the field for
the remote port and address (DPORT, DADDR) differs from the tool
itself (RPORT, RADDR). This patch fix the man pages.

The man page of tcpdrop mentions options that do not exists. This
patch fix that. It also fix the comment at the head of the code that
too mentions inexistents options.

The man pages for solisten and sofdsnoop are missing. This patch adds
them.

The deadlock_detector tools has been rename simply deadlock, but its
manpage is still named deadlock_detector although its content has been
updated. This patch rename the manpage.

### CODE REVIEW

File: man/man8/sofdsnoop.8 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+trace this TID only
+.TP
+\fB\-n\fR NAME, \fB\-\-name\fR NAME
+only print process names containing this name
+.TP
+\fB\-d\fR DURATION, \fB\-\-duration\fR DURATION
+total duration of trace in seconds
+.SH EXAMPLES
+.TP
+Trace file descriptors passes
```
Comment (yonghong-song) [inline man/man8/sofdsnoop.8]: Maybe "Trace passed file descriptors"?

File: man/man8/solisten.8 [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.TH SOLISTEN 8 "2019-07-29" "USER COMMANDS"
+.SH NAME
+solisten \- Trace listening socket
+.SH SYNOPSIS
+usage: solisten [\-h] [\-\-show\-netns] [\-p PID] [\-n NETNS]
+.SH DESCRIPTION
+All IPv4 listen attempts are traced, even if they ultimately fail or the
```
[the remark this answers is not in the document]
Comment (jeromemarchand) [reply man/man8/solisten.8]: Sure. I got that sentence from somewhere else, the example file or the usage message, maybe both, I don't remember. I'll fix it there too.

File: man/man8/sofdsnoop.8 (hunk 1 above)
Comment (jeromemarchand) [reply man/man8/sofdsnoop.8]: That doesn't sound like something I wrote. It probably comes from the usage message (I used help2man to draft the man page). I'll check and fix it there too if needed.

File: man/man8/solisten.8 (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeromemarchand) [reply man/man8/solisten.8]: Actually, it was from a comment at the head of the source file.

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Comment by yonghong-song** [comment]
LGTM. Thanks.

**Pull request merged**